apps/gro/views/address-outside.html

```diff
@@ -0,0 +1,54 @@
+{{<layout}}
+{{$step}}Step 2{{/step}}
+
+{{$journeyHeader}}
+{{#t}}journey.header{{/t}}
+{{/journeyHeader}}
+
+{{$validationSummary}}
+{{> partials-validation-summary}}
+{{/validationSummary}}
+
+{{$propositionHeader}}
+{{> partials-navigation}}
+{{/propositionHeader}}
+
+{{$header}}{{#t}}pages.post.header{{/t}}{{/header}}
+
+{{$content}}
+
+{{<partials-form}}
+
+{{$form}}
+
+{{#input-text}}address-text-one{{/input-text}}
+
+
+<legend class=visuallyhidden>
+  <span class="form-label-bold">{{#t}}fields.address-text-two.label{{/t}}</span>
+</legend>
+{{#input-text}}address-text-two{{/input-text}}
+
+<legend class=visuallyhidden>
+  <span class="form-label-bold">{{#t}}fields.address-text-three.label{{/t}}</span>
+</legend>
```

Comment:
This is for accessibility for screen readers. At some stage create a template mixin?